lib/tasks/publish.rake

```diff
@@ -0,0 +1,13 @@
+# This tasks will publish documents that are in a draft publication state.
+
+require "services"
+
+namespace :publish do
+  desc "Publishes all documents in a draft publication state.\n" \
+    "Usage: `rake publish:all` or with types `rake publish:all protected_food_dring_name esi_fund`"
```

Comment:
```suggestion
    "Usage: `rake publish:all` or with types `rake publish:all[protected_food_drink_name esi_fund]`"
```

Without that it actually ends up publishing everything and then trying to run a task called `protected_food_drink_name`. I'd actually suggest that we make it not possible to run this task and publish everything, because it's highly unlikely we'd ever want to do that and has a risk that someone does it accidentally.

Reply:
Glad you said that :D I actually added the ability to call it on a specific set of types, because the original requirement was to publish all content with one task regardless! And I asked twice.  
Will remove the ability to do that now :)